cmd/buildah/pull.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+	"github.com/projectatomic/buildah/pkg/parse"
+	util "github.com/projectatomic/buildah/util"
+	"github.com/urfave/cli"
+)
+
+var (
+	pullFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "authfile",
+			Usage: "path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
+		},
+		cli.StringFlag{
+			Name:  "cert-dir",
+			Value: "",
+			Usage: "use certificates at the specified path to access the registry",
+		},
+		cli.StringFlag{
+			Name:  "creds",
+			Value: "",
+			Usage: "use `[username[:password]]` for accessing the registry",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "don't output progress information when pulling images",
+		},
+		cli.StringFlag{
+			Name:  "shm-size",
+			Usage: "size of '/dev/shm'. The format is `<number><unit>`.",
+			Value: "65536k",
+		},
+		cli.StringFlag{
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
+		},
+		cli.BoolTFlag{
+			Name:  "tls-verify",
+			Usage: "require HTTPS and verify certificates when accessing the registry",
+		},
+	}
+	pullDescription = "Pulls an image from the specified location"
```

Comment:
How about we change this to.

Pulls and image from the specified registry or one of the registries listed in %s if no registry was specified and stores it in 
container storage in the %s directory.

Reply:
podman pull says

```
Pulls an image from a registry and stores it locally.
An image can be pulled using its tag or digest. If a tag is not
specified, the image with the 'latest' tag (if it exists) is pulled.
```

Reply:
Done